easyDiffractionApp/Gui/Pages/Project/qmldir

```diff
@@ -1,5 +1,7 @@
 module Project
 
+AboutDialog 1.0 AboutDialog.qml
```

Comment:
AboutDialog.qml file seems to be missing in Gui/Pages/Project

Reply:
Embarrassing...

Reply:
Fixed. Sorry